File: src/AAS_Cores/OPC_UA_AAS_Core/src/utilities/AASarchiveInfo.py
class AASarchiveInfo:
    """This class contains all information about the AAS Archive paths."""

    # Paths related to interactions between Manager and Core
    INTERACTIONS_FOLDER_PATH = '/aas_archive/interactions'
    CORE_INTERACTIONS_FOLDER_PATH = '/aas_archive/interactions/core'
    MANAGER_INTERACTIONS_FOLDER_PATH = '/aas_archive/interactions/manager'

    # Subpath of interaction files
    SVC_REQUEST_FILE_SUBPATH = '/svcRequests.json'
    SVC_RESPONSE_FILE_SUBPATH = '/svcResponses.json'

    # ConfigMap files
    CONFIG_MAP_PATH = '/aas_archive/config'
    CM_AAS_PROPERTIES_FILENAME = 'aas.properties'
    CM_ASSET_PROPERTIES_FILENAME = 'asset.properties'
    CM_SM_PROPERTIES_FILENAME = 'submodels.properties'

    # Status file
    MANAGER_STATUS_FILE_PATH = '/aas_archive/status/aas_manager.json'
    CORE_STATUS_FILE_PATH = '/aas_archive/status/aas_core.json'

File: src/AAS_Cores/logical_asset_cores/WarehouseTransport_AAS_Core/src/main/java/information/AAS_ArchiveInfo.java
package information;

public class AAS_ArchiveInfo {
    /**
     * This class will contain all information about the AAS Archive.
     */

    // Paths related to interactions between AAS Core and AAS Manager
    public static final String coreInteractionsFolderPath = "/aas_archive/interactions/core";
    public static final String managerInteractionsFolderPath = "/aas_archive/interactions/manager";

    // Subpath of interaction files
    public static final String svcRequestFileSubPath = "/svcRequests.json";
    public static final String svcResponseFileSubPath = "/svcResponses.json";


    // Status file
    public static final String coreStatusFilePath = "/aas_archive/status/aas_core.json";
    public static final String managerStatusFilePath = "/aas_archive/status/aas_manager.json";

    // Config file
    public static final String configFolderPath = "/aas_archive/config";
    public static final String aasPropertiesFileName = "aas.properties";

}
